Tests/SwiftParserTest/LexerTests.swift

```diff
@@ -1080,6 +1080,41 @@ public class LexerTests: ParserTestCase {
     }
   }
 
+  func testUTF16Surrogates_1() {
```

Comment:
Nitpick: In other placed of the codebase where we have two similar test cases that are just numbered, we don’t use underscores.

```suggestion
  func testUTF16Surrogates1() {
```

Same for the `_2` test case below.

Reply:
I might have been confused by the test case named `testInvalidUtf8_3`, but I believe it's the only exception.
I changed its name to not use an underscore.
@ahoppen